bcs/bcs.go

```diff
@@ -178,8 +185,8 @@ func DeserializeMapToSlices[K, V any](bcs *Deserializer) (keys []K, values []V)
 	return
 }
 
-// BcsSerialize serializes a single item
-func BcsSerialize(value BCSStruct) (bcsBlob []byte, err error) {
+// Serialize serializes a single item
+func Serialize(value Marshaler) (bcsBlob []byte, err error) {
```

Comment:
Shouldn't this be... marshalable?

Reply:
(and we are not the first to notice that this part of stdlib grammar reads a little weird to English speakers, but someone important at Go prefers -er to -able)